auslib/web/admin/swagger/api.yaml

```diff
@@ -841,7 +874,36 @@ paths:
                 type: array
                 minItems: 0
                 items:
-                  $ref: '#/definitions/ReleaseRevision'
+                  type: object
```

Comment:
Previously I had defined 'ReleaseRevision' model to define the object as element for every item in revisions array above. However, I realized that the ReleaseRevision model won't re-used at all in any other API, hence I moved it out from definitions.